example/gardener-local/gardenlet/values-kind-ha.yaml

```diff
@@ -0,0 +1,7 @@
+global:
+  gardenlet:
+    config:
+      seedConfig:
+        metadata:
+          labels:
+            seed.gardener.cloud/multi-zonal: "true"
```

Comment:
Now that #6723 is merged, instead of using the label can you use `.spec.highAvailability`?